DanielVargas/tic_tac_toe/src/com/company/Tictactoe.java

```diff
@@ -0,0 +1,176 @@
+package com.company;
+import java.util.Scanner;
+
+public class Tictactoe {
+    final int cols=3;
+    final int rows=3;
+    private String [][] board;
+    private String player1;
+    private String player2;
+    private int num_of_plays;
+    Tictactoe(String player1, String player2) {
+        this.board = new String[this.cols][this.rows];
+        this.player1 = player1;
+        this.player2 = player2;
+        this.num_of_plays=0;
+        for (int i = 0; i < this.rows; i++) {
+            for (int j = 0; j < this.cols; j++) {
+                this.board[i][j] = "";
+            }
+        }
+    }
```

Comment:
Please maintain the same convention for code blocks in all your project, in Java the convention is to put the opening curly brace in the same line the class/method is declared:
```java
public class Foo {
    public static main void(String[] args) {
    }
}
```
If you're using Intellij you can make use of this [tool](https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html) to solve this easily.